.github/workflows/ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
           curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL "https://sh.rustup.rs" | sh -s -- --profile minimal --default-toolchain none -y
           echo "${CARGO_HOME:-$HOME/.cargo}/bin" >> $GITHUB_PATH
       - name: Install SQLFluff
-        run: pip install sqlfluff==2.3.0
+        run: pip install sqlfluff==2.3.5
```

Comment:
don't know where else to put it: Latest rust stable is now 1.75, do you want to update it, because the card has a success criterion to be latest stable?

Reply:
Sure, but I did already update it: [27b766f](https://github.com/geo-engine/geoengine/pull/913/commits/27b766f134b6798d5f524722626efa2fa8190839)